src/init/application/application.ts

```diff
@@ -75,6 +75,8 @@ export function createApplicationGenerator(templateName: string):
       ];
 
       this.props = await this.prompt(prompts);
+      this.props.elementClassName = this.props.elementName.replace(
+          /(^|\W)+(.)/g, (_match, _p0, p1) => p1.toUpperCase());
```

Comment:
I think we should use the same matcher as core polymer: `replace(/-[a-z]/g , (m) => m[1].toUpperCase())`